Quality of Life improvements

moshthepitt opened the pull request:
- Add [release drafter](https://github.com/apps/release-drafter)
- Add tags for git and pgpass tasks
- Update python requirements
- Fix ansible-lint errors

moshthepitt committed:
Add release drafter

moshthepitt committed:
Add helpful tags

moshthepitt committed:
Fix linter errors

moshthepitt committed:
Ask ansible-lint to skip rul 106

This rule specifies: Role name {} does not match ``^[a-z][a-z0-9_]+$`` pattern'

We are skipping it instead of changing the role name.

moshthepitt committed:
Update python requirements

DavisRayM commented on tasks/setup.yml:
```diff
@@ -27,6 +27,7 @@
     owner: "{{ sqitch_migrations_system_user }}"
     group: "{{ sqitch_migrations_system_group }}"
     path: "{{ item }}"
+    mode: '0644'
```
A bit nit-picky but should we use double-quotation marks here (same as in line 43)?

moshthepitt replied:
Yes will do

moshthepitt committed:
Use tagged version

moshthepitt committed:
Clean up

moshthepitt committed:
Remove cron verification tasks

Why?

1.  These tests are finnicky and we spend a lot of energy to make them work.
2.  However, these tests are not necessary from the point of vie of this role.
    We don't need to test that the cron service works - that is out of scope.
    We just need to ensure that we have the correct cron configuration, and we
    already do that.

moshthepitt committed:
Update packages

moshthepitt committed:
Lock down ansible versions

moshthepitt committed:
Remove MOLECULE_EPHEMERAL_DIRECTORY

moshthepitt committed:
Fix ansible versions
